watertap/unit_models/nanofiltration_DSPMDE_0D.py

```diff
@@ -0,0 +1,1424 @@
+                units=units_meta('length'),
+                doc='Effective membrane length')
+            self.width = Var(
+                initialize=5,
+                bounds=(0.1, 5e2),
+                domain=NonNegativeReals,
+                units=units_meta('length'),
+                doc='Effective feed-channel width')
+            self.channel_height = Var(
+                initialize=1e-3,
+                domain=NonNegativeReals,
+                bounds=(1e-4, 5e-3),
+                units=units_meta('length'),
+                doc='Feed channel height')
+            self.spacer_porosity = Var(
+                initialize=0.95,
+                bounds=(0.1, 1.001),
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc='Feed-channel spacer porosity')
+            self.velocity = Var(
+                self.flowsheet().config.time,
+                self.io_list,
+                initialize=0.5,
+                bounds=(1e-3, 2),
+                domain=NonNegativeReals,
+                units=units_meta('length') / units_meta('time'),
+                doc="Crossflow velocity in feed channel at inlet and outlet")
+            self.N_Sc_comp = Var(
+                self.flowsheet().config.time,
+                self.io_list,
+                solute_set,
+                initialize=5e2,
+                bounds=(1e2, 2e3),
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="Schmidt number at inlet and outlet")
+            self.N_Pe_comp = Var(
+                self.flowsheet().config.time,
+                self.io_list,
+                solute_set,
+                initialize=1e5,
+                bounds=(5e3, None),  # TODO:unsure of value ranges at the moment
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="Peclet number at inlet and outlet")
+            self.spacer_mixing_efficiency = Var(
+                initialize=0.5,
+                # bounds=(1e2, 2e3),  # TODO:unsure of value ranges at the moment- since this is efficiency, assuming 0 -1
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="Mixing efficiency of spacer net")
+            self.spacer_mixing_length = Var(
+                initialize=0.6,
+                # bounds=(1e2, 5e3),  #TODO:unsure of value ranges at the moment
+                domain=NonNegativeReals,
+                units=units_meta('length'),
+                doc="Characteristic length of spacer")
+        ###############################################################################################################
+        # Expressions
+        ###############################################################################################################
+        # Make expressions that don't depend on any variables
+        self._make_expressions()
```

Comment:
This is a minor thing, but I'm a bit confused by the existence of this method alongside the Expressions below.

Reply:
I'm going to clean this up in the next PR. Essentially I was toggling between Expressions and Vars with Constraints while troubleshooting the model.